### PR TITLE
New package: StreetPea.chiaki-ng version 1.9.1

### PR DESCRIPTION
**Pull request by streetpea**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/190343)

### CODE REVIEW

**Comment by streetpea** [comment]
@microsoft-github-policy-service agree

File: manifests/s/StreetPea/chiaki-ng/1.9.1/StreetPea.chiaki-ng.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: StreetPea.chiaki-ng
+PackageVersion: 1.9.1
+InstallerType: inno
+Installers:
```
Comment (stephengillie) [inline manifests/s/StreetPea/chiaki-ng/1.9.1/StreetPea.chiaki-ng.installer.yaml:7]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Manual Validation ended with:
image

**Comment by stephengillie** [comment]
Close with reason: Duplicate of #190246;

**Comment by streetpea** [comment]
@stephengillie that looks like a completely different project from the link.

**Comment by stephengillie** [comment]
Hi @streetpea,

Apologies for accientally closing this PR. I meant to close another as duplicate, and got this PR number by mistake as I'm moving too fast.

**Comment by stephengillie** [comment]
HI @streetpea,

It appears that the application has another dependency:
image

Could `KhronosGroup.VulkanSDK` be the correct dependency?  

Once a dependency is installed, it usually remains installed for some time. Most uninstallers do not uninstall dependencies, and the package manager does not. So if you already had the package installed on your system, the dependencies would have been installed some time ago, and your system wouldn't present any of these error messages. This is part of why validations are performed on freshly-installed OSes.

**Comment by streetpea** [comment]
@stephengillie it seems that that dependency is installed with graphics drivers which is probably why I haven't heard anybody complain about it. I added it.

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> Faulting application name: chiaki.exe, version: 1.9.1.0, time stamp: 0x6731a7cb 
> Faulting module name: chiaki.exe, version: 1.9.1.0, time stamp: 0x6731a7cb Exception code: 0xc0000005 Fault offset: 0x000000000005db0e 
> Faulting process id: 0x25bc 
> Faulting application start time: 0x01db3561e649b654 
> Faulting application path: C:\Program Files\chiaki-ng\chiaki.exe 
> Faulting module path: C:\Program Files\chiaki-ng\chiaki.exe Report Id: b294148e-305f-420c-a82f-559afcc7e905 
> Faulting package full name:  
> Faulting package-relative application ID:   Fault bucket 1571553288640585701, type 4 Event Name: APPCRASH Response: Not available Cab Id: 0  Problem signature: P1: chiaki.exe P2: 1.9.1.0 P3: 6731a7cb P4: chiaki.exe P5: 1.9.1.0 P6: 6731a7cb P7: c0000005 P8: 000000000005db0e P9:  P10:   Attached files: \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERFA4D.tmp.dmp \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERFB19.tmp.WERInternalMetadata.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERFB2A.tmp.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERFB3F.tmp.csv \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERFB50.tmp.txt  These files may be available here: \\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_chiaki.exe_441459e35264f7e28f20536c483284c5ba995bd8_0e021de8_6a93495e-5199-434c-9724-e5b7ccdbbeeb  Analysis symbol:  Rechecking for solution: 0 Report Id: b294148e-305f-420c-a82f-559afcc7e905 Report Status: 268435456 Hashed bucket: c0989a27d960fa0c35cf4763bb04dfe5 Cab Guid: 0

(Automated response - build 897.)

**Comment by stephengillie** [comment]
> @ stephengillie it seems that that dependency is installed with graphics drivers which is probably why I haven't heard anybody complain about it. I added it. Thank you for your patience.

It's my pleasure to help. The dependency might be missing from the VMs as they have inferior graphics drivers - if it's not necessary in most PCs then it should probably be removed. 

Manual Validation ended with the errors above being logged in the Event Log.  The application window didn't open - I don't have more info about why, but if you have any troubleshooting suggestions, I'd be happy to try them.

**Comment by streetpea** [comment]
@stephengillie is it possible to run debugview https://learn.microsoft.com/en-us/sysinternals/downloads/debugview on the system and see if any logs pop up in there?

**Comment by streetpea** [comment]
@stephengillie according to Khronos who makes Vulkan, you shouldn’t include Vulkan-1.dll with your application and should install allow the drivers to update it or users to manually update it if their drivers are too old. https://github.com/KhronosGroup/Vulkan-Loader/blob/main/docs/LoaderApplicationInterface.md#bundling-the-loader-with-an-application

**Comment by stephengillie** [comment]
Apologies for the delay in responding - I'm dealing with capacity issues. Yes, feel free to remove the dependency. I am unsure about getting debugview onto the VM - does it have a `PackageIdentifier` in the repo?

**Comment by streetpea** [comment]
@stephengillie you can download the exe for debugview directly from sysinternals using this link https://live.sysinternals.com/Dbgview.exe with curl or wget

**Comment by stephengillie** [comment]
Unfortunately, it didn't find any logs on the VM. 
image

**Comment by streetpea** [comment]
Hmm. Maybe it needs administrative access to run based on that popup. If you start debug view first and then leave it open when launching chiaki-ng various logs should appear in debug view.

**Comment by streetpea** [comment]
@stephengillie did you open DebugView first and then leave it open when launching chiaki-ng?

**Comment by stephengillie** [comment]
I haven't used DebugView before, sorry. This seems to be a problem with the VM driver:
image

- https://github.com/microsoft/winget-pkgs/issues/107639
- https://github.com/microsoft/winget-pkgs/issues/12906

**Review by stephengillie — APPROVED** [review]

**Pull request merged**